compiler/rustc_mir/src/transform/const_prop.rs

```diff
@@ -708,7 +709,7 @@ impl<'mir, 'tcx> ConstPropagator<'mir, 'tcx> {
             return None;
         }
 
-        if self.tcx.sess.opts.debugging_opts.mir_opt_level >= 3 {
+        if self.tcx.sess.opts.debugging_opts.mir_opt_level.unwrap_or(MIR_OPT_LEVEL_DEFAULT) >= 3 {
```

Comment:
It looks like I can add functions to the `DebuggingOptions` struct, so I could replace what was:
```rust
    self.tcx.sess.opts.debugging_opts.mir_opt_level
```
and is now:
```rust
self.tcx.sess.opts.debugging_opts.mir_opt_level.unwrap_or(MIR_OPT_LEVEL_DEFAULT)
```
with:
```rust
self.tcx.sess.opts.debugging_opts.get_mir_opt_level()
```
which would also avoid the need to `use` MIR_OPT_LEVEL_DEFAULT.

Reply:
I'm making this change in a new commit